apps/pie-docs/src/_includes/content-page.njk

```diff
@@ -18,26 +18,16 @@
         {% include "page-tabs.njk" %}
     {% endif %}
 
-    {# temporarily adding a condition to check if the title is PIE Design System - this allows an empty home page not to display the coming soon graphic in the page body #}
-    {% if content %}
-        <div class="c-content">
-            <div class="c-content-container">
-                {% if shouldShowContents %}
-                    <aside>
-                        <h2>Contents</h2>
-                        {{ content | toc | safe}}
-                        <hr />
-                    </aside>
-                {% endif %}
-                {{ content | safe }}
-            </div>
-        </div>
-    {% else %}
-        <div class="c-content u-noPad">
```

Comment:
I've removed this because it didn't really make sense. There is little reason to add an empty page tab and this logic made it possible to add misleading coming soon content to the site